requirements.txt

```diff
@@ -16,6 +16,7 @@ https://github.com/LyridInc/graphene-django-cud/tarball/master
 phonenumbers==8.12.42
 python-dotenv==0.19.2
 requests==2.27.1
+requests-oauthlib==1.3.1
```

Comment:
Is this a requirement for the twitter login to function properly?

Reply:
Nope, actually, I think using requests should work. I'll try changing it to use the requests library if that makes the code cleaner.

Reply:
Okay so I just checked and social_core library uses the request_oauthlib too, so I'm removing this line since it should be automatically already inside the venv.